packages/data-stores/src/wpcom-plans-ui/selectors.ts

```diff
@@ -1,3 +1,7 @@
 import type { State } from './reducer';
 
 export const isDomainUpsellDialogShown = ( state: State ) => !! state.showDomainUpsellDialog;
+export const getStorageAddOnForPlan = ( state: State ) => ( plan: string ) => {
+	const selectedStorageAddOnsForPlans = state?.selectedStorageAddOnsForPlans;
```

Comment:
Can `state` be undefined?
and is there a need for the intermediate variable/assignment?

Reply:
Ah also a good catch. It's selectedStorageAddOnsForPlans that initializes as `undefined`, not `state` itself.

I added the intermediate variable for readability, but no strong opinions. I'll remove it since it seems like it might not be helping here though 👍